govc/host/service/ls.go

```diff
@@ -110,7 +110,7 @@ type optionResult []types.HostService
 func (services optionResult) Write(w io.Writer) error {
 	tw := tabwriter.NewWriter(w, 2, 0, 2, ' ', 0)
 
-	fmt.Fprintf(tw, "%s\t%s\t%v\t%s\n", "Key", "Policy", "Status", "Label")
+	fmt.Fprintf(tw, "%s\t%s\t%v\t%s\n", "Key", "policy", "Status", "Label")
```

Comment:
Looks like this was s///, can you revert this change?

Reply:
Oops, thanks for catching!